docs/src/modules/components/AppTableOfContents.js

```diff
@@ -36,16 +36,17 @@ const useStyles = makeStyles((theme) => ({
   },
   item: {
     fontSize: 13,
-    padding: theme.spacing(0.5, 0, 0.5, 1),
```

Comment:
What about restoring this style? There is an issue with the spacing between the items. There isn't enough space to understand to read and understand how the text is grouped when it wraps.

Reply:
If we're interested in readability I went with line-height and also used `rem` instead of a fixed size. Before we couldn't scale the readability.